src/category_theory/limits/shapes/equalizers.lean

```diff
@@ -62,13 +63,21 @@ instance fintype_walking_parallel_pair : fintype walking_parallel_pair :=
 open walking_parallel_pair
 
 /-- The type family of morphisms for the diagram indexing a (co)equalizer. -/
-inductive walking_parallel_pair_hom : walking_parallel_pair → walking_parallel_pair → Type v
+@[derive _root_.decidable_eq] inductive walking_parallel_pair_hom :
+  walking_parallel_pair → walking_parallel_pair → Type v
 | left : walking_parallel_pair_hom zero one
 | right : walking_parallel_pair_hom zero one
 | id : Π X : walking_parallel_pair.{v}, walking_parallel_pair_hom X X
 
 open walking_parallel_pair_hom
 
+instance (j j' : walking_parallel_pair) : fintype (walking_parallel_pair_hom j j') :=
```

Comment:
I would really prefer to see a derive handler that can cope with this. Do you know why the existing one doesn't work here? (I haven't looked at all.)

Reply:
When I put `attribute [derive fintype] walking_parallel_pair_hom`, I get `failed to find a derive handler for 'fintype'`. Since even for object types like `walking_parallel_pair` the fintype instance is implemented manually, I concluded that there simply is no derive handler for `fintype`. A grep over the mathlib repository seems to support that conclusion.

Reply:
It seems the only other successful `@[derive fintype]` is in `archive/sensitivity.lean`, where `delta_instance` is all that's required to get it done. A general purpose `derive_handler` for `fintype` unfortunately doesn't exist, but this seems like the sort of thing @cipher1024 likes. :-)